package.json

```diff
@@ -21,6 +21,24 @@
   ],
   "main": "./out/extension.js",
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "ONE-Compiler",
+          "title": "ONE Compiler",
```

Comment:
It's only my preference, but how about just `ONE`? :)

Reply:
Fixed.

Reply:
![image](https://user-images.githubusercontent.com/63768207/157812495-2b347a11-772b-4189-91d6-062f7b79410d.png)